server.js

```diff
@@ -0,0 +1,21 @@
+/*eslint-disable*/
+const express = require("express");
+const webpack = require("webpack");
+const webpackDevMiddleware = require("webpack-dev-middleware");
+
+const app = express();
+const config = require("./webpack.config.js");
+const compiler = webpack(config);
+
+// Tell express to use the webpack-dev-middleware and use the webpack.config.js
+// configuration file as a base.
+app.use(
+  webpackDevMiddleware(compiler, {
+    publicPath: config.output.publicPath,
+  })
+);
+
+// Serve the files on port 3000.
+app.listen(3000, function () {
+  console.log("Example app listening on port 3000!\n");
+});
```

Comment:
- [ ] Kindly make sure you remove the `console.log()` from your code base as this is ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.